src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -171,12 +170,37 @@ private static String getTokenFromCache(String userName) {
       }
     } catch (IOException e) {
       PageObjectLogging.log(IOEXCEPTION_COMMAND,
-                            IOEXCEPTION_ERROR_MESSAGE + ExceptionUtils.getStackTrace(e), false);
+          IOEXCEPTION_ERROR_MESSAGE + ExceptionUtils.getStackTrace(e), false);
       throw new WebDriverException(e);
     }
     return "";
   }
 
+  public static String getUserId(String userName) {
+    String getUserIDURL = "";
+    try {
+      getUserIDURL = String.format("%s/api.php?action=query&list=users&ususers=%s&format=json",
+          new UrlBuilder().getUrlForWiki(), URLEncoder.encode(userName, "UTF-8"));
+    } catch (UnsupportedEncodingException e) {
+      PageObjectLogging.logError("UNSUPPORTED ENCODING EXCEPTION", e);
+      throw new WebDriverException(e);
+    }
+
+    HttpGet httpGet = new HttpGet(getUserIDURL);
+
+    try {
+      String userId = getDefaultClient().execute(httpGet, extractUserId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "userId". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1488)